packages/lambdas/src/fhir-to-medical-record.ts

```diff
@@ -45,21 +42,18 @@ export const handler = Sentry.AWSLambda.wrapHandler(
     const { log } = out(`cx ${cxId}, patient ${patientId}`);
     log(
       `Running with conversionType: ${conversionType}, dateFrom: ${dateFrom}, ` +
-        `dateTo: ${dateTo}, fileName: ${fhirFileName}`
+        `dateTo: ${dateTo}, fileName: ${fhirFileName}, bucket: ${bucketName}}`
     );
 
-    const bundle = await getBundleFromS3(fhirFileName);
-    log(`Bundle: ${JSON.stringify(bundle)}`);
-
     try {
       const log = prefixedLog(`patient ${patientId}`);
 
-      if (isSandbox) return processSandbox({ firstName, conversionType });
-
       if (!converterKeysTableName) {
         throw new Error(`Programming error - SIDECHAIN_FHIR_CONVERTER_KEYS_TABLE_NAME is not set`);
```

Comment:
Can't we just use `getEnvOrFail` to get this env var instead and always have it as string at this point, so no need for this check?